### PR TITLE
unnecessary use of url for localhost in doc for service_to_service.rst file

### PR DESCRIPTION
**Pull request by ankatare**
Signed-off-by: Abhay Narayan Katare <abhay.katare@india.nec.com>

Commit Message: Unnecessary use of url in doc
Additional Description: NA
Risk Level: LOW
Testing: 
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: docs/root/intro/deployment_types/service_to_service.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ Service to service egress listener
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This is the port used by applications to talk to other services in the infrastructure. For example,
-http://localhost:9001. HTTP and gRPC requests use the HTTP/1.1 *host* header or the HTTP/2 or HTTP/3
+*http://localhost:9001*. HTTP and gRPC requests use the HTTP/1.1 *host* header or the HTTP/2 or HTTP/3
```
Comment (phlax) [inline docs/root/intro/deployment_types/service_to_service.rst:17]: im not sure what the problem with the link is, but i dont think italicizing it helps any

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: @phlax Hi, Thanks for review. My Point is, placing URLs here will create confusion as it is not configuration sections where user can use it while configuring it. Moreover also it also helps to view in consistence similar to below section as https://www.envoyproxy.io/docs/envoy/latest/intro/deployment_types/service_to_service#service-to-service-ingress-listener

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: for  italicizing, i just checked section https://www.envoyproxy.io/docs/envoy/latest/intro/deployment_types/service_to_service#service-to-service-ingress-listener and its source code for removing urls. We can just remove urls without  italicizing etc.

Comment (phlax) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: im trying to get rid of the use of italics, its one of the things that has made the docs really hard to read

not sure about removal, but if you want to update the PR to remove, im happy to read the docs through properly

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: Ok.

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: @phlax just on the closure side of this PR. Would you please help me understand what is the meaning of read the docs here ?

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: is that https://readthedocs.org/  ? Do we have readthe docs for envoyproxy as well and if yes, how do we do code changes here ?

Comment (phlax) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: we use the RTD (readthedocs) theme, we dont host with them tho

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: Ok. Got it. Thanks for quick response

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: Before reaching to conclusion codewise, i would like to highlight below URLs of ONAP community, They follow RTD. please suggest if it is ok to take as example here as well.
https://docs.onap.org/en/latest/guides/onap-user/onap-portal-user/onap-portal-user.html#access

Comment (phlax) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: as i said, there is no benefit to using italics, it is semantically wrong and confusing

italics are for emphasis, using it elsewhere renders that meaningless and does nothing for code or urls

i think the best thing is to leave this as it is

Comment (ankatare) [reply docs/root/intro/deployment_types/service_to_service.rst:17]: Okay.

**Comment by ankatare** [comment]
Closing this

**Pull request closed without merging**